plugins/catalog-react/src/types.ts

```diff
@@ -23,7 +23,10 @@ export type EntityFilter = {
    *   { field: 'kind', values: ['component'] }
    *   { field: 'metadata.name', values: ['component-1', 'component-2'] }
    */
-  getCatalogFilters?: () => Record<string, string | string[]>;
+  getCatalogFilters?: () => Record<
+    string,
+    string | symbol | (string | symbol)[]
```

Comment:
Shouldn't need `symbol[]` really, right? 🤔  Or does it make the typing annoying if you leave it out?

Reply:
hm this was done to match: https://github.com/backstage/backstage/blob/8f5befa35eff39a9ea3cf062ae93cab78f3dd3fa/packages/catalog-client/src/types/api.ts#L24 which was required for https://github.com/backstage/backstage/pull/6444 since it made typing annoying if it wasn't included